### DISCUSSION TITLE
New Roadmap

### DISCUSSION THREAD

**Discussion by Lucia-Fonseca** [skypyproject/skypy#532] [Ideas]
To explore new directions including features from radio, gravitational waves and optical astronomy.

**Comment by itrharrison** [comment]
Part of the idea here is to lower the threshold of what is considered 'mature' for a new module to be merged into SkyPy.

I would advocate for this level of maturity being that a module contains at least one function which creates either a new SkyPy catalogue or new columns in a SkyPy catalogue, with all of the functions being subject to the same requirements in terms of documents and unit tests etc as before.

Hopefully this will encourage contributions from broader communities, by providing them with a visible place to contribute, as opposed to the current immediate image which is of SkyPy as 'only a galaxies thing'.
